stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -77,8 +76,6 @@ public interface ChatDomain {
     /** The retry policy for retrying failed requests */
     public var retryPolicy: RetryPolicy
 
-    public val typingUpdates: LiveData<TypingEvent>
```

Comment:
Talking with designers, it is not totally dropped from our new design.
On the other hand, it is a public property that we expose in our SDK, we shouldn't break our public API because we don't use it into our new design

Reply:
@leandroBorgesFerreira why did you add this? was it for a customer or for the new design?

for our design the right way would be to expose a channel preview class that contains this information at the queryChannelController level.